Dockerfile

```diff
@@ -0,0 +1,61 @@
+# translate/pootle
+#
+# VERSION	0.0.1
+
+FROM debian
+
+MAINTAINER Ryan Northey <ryan@synca.io>
+
+RUN apt-get update && \
+    apt-get install -y \
+            python-pip \
+	    libxml2-dev \
+	    libxslt-dev \
+	    python-dev \
```

Comment:
because that's what the current debian:latest/stable/jessie provides

we could  put ```FROM debian:jessie``` on line 5